### PR TITLE
shader/translator: Add null spir-v check guards.

### PR DESCRIPTION
**Pull request by Zangetsu38**
- Add all spir-v check guard if shader value don't success loaded, prevent crashing, normally for lot case.
- Add log error for can debug it in futur and probably fix/improve render.
- Run clang format.

### CODE REVIEW

File: src/emulator/shader/src/translator/alu.cpp [hunk 1]
```diff
@@ -127,6 +127,7 @@ bool USSETranslatorVisitor::vmad(
     spv::Id vsrc2 = load(inst.opr.src2, write_mask, 0);
 
     if (vsrc0 == spv::NoResult || vsrc1 == spv::NoResult || vsrc2 == spv::NoResult) {
+        LOG_ERROR("Source Not Loaded!(vsrc0: {}, vsr1: {}, vsrc2: {})", vsrc0, vsrc1, vsrc2);
```
Comment (VelocityRa) [inline src/emulator/shader/src/translator/alu.cpp]:
```suggestion
        LOG_ERROR("Source not loaded (vsrc0: {}, vsr1: {}, vsrc2: {})", vsrc0, vsrc1, vsrc2);
```
same for the other messages, only capitalize the first word

File: src/emulator/host/src/host.cpp [hunk 2]
```diff
@@ -27,7 +27,7 @@
 #include <unordered_set>
 
 #ifdef NDEBUG // Leave it as non-constexpr on Debug so that we can enable/disable it at will via set_log_import_calls
-    constexpr
+constexpr
```
Comment (VelocityRa) [inline src/emulator/host/src/host.cpp]: Why was this changed? I have already run clang-format for this file. Make sure yours is using our `.clang-format`.

Comment (Zangetsu38) [reply src/emulator/host/src/host.cpp]: i have just run format.bat like all time before so no idea

File: src/emulator/shader/src/translator/alu.cpp (hunk 1 above)
Comment (Zangetsu38) [reply src/emulator/shader/src/translator/alu.cpp]: ah yes sorry

File: src/emulator/host/src/host.cpp (hunk 2 above)
Comment (Zangetsu38) [reply src/emulator/host/src/host.cpp]: i have revert it

Comment (VelocityRa) [reply src/emulator/host/src/host.cpp]: I re-run it and it did the same... Not sure what's up, re-run it and include all changes it does to your commit.

Comment (Zangetsu38) [reply src/emulator/host/src/host.cpp]: oki, is done

File: src/emulator/shader/src/translator/data.cpp [hunk 3]
```diff
@@ -201,6 +206,11 @@ bool USSETranslatorVisitor::vmov(
     spv::Id source = load(inst.opr.src1, dest_mask, src1_repeat_offset);
     store(inst.opr.dest, source, dest_mask, dest_repeat_offset);
 
+    if (source == spv::NoResult) {
+        LOG_ERROR("Source Not Loader!");
```
Comment (VelocityRa) [inline src/emulator/shader/src/translator/data.cpp]: fix this
also remove `!` from all messages and add a space
and you forgot closing `)` in some of them

Comment (Zangetsu38) [reply src/emulator/shader/src/translator/data.cpp]: fix this ? what is this ?

Comment (Zangetsu38) [reply src/emulator/shader/src/translator/data.cpp]: all is done

File: src/emulator/shader/src/translator/alu.cpp [hunk 4]
```diff
@@ -127,6 +127,7 @@ bool USSETranslatorVisitor::vmad(
     spv::Id vsrc2 = load(inst.opr.src2, write_mask, 0);
 
     if (vsrc0 == spv::NoResult || vsrc1 == spv::NoResult || vsrc2 == spv::NoResult) {
+        LOG_ERROR("Source not loaded (vsrc0: {}, vsr1: {}, vsrc2: {})", vsrc0, vsrc1, vsrc2);
```
Comment (pent0) [inline src/emulator/shader/src/translator/alu.cpp]: No point printing sources out. You can just do
```suggestion
        LOG_ERROR("Source not loaded!");
```

Same with others

Comment (pent0) [reply src/emulator/shader/src/translator/alu.cpp]: Also, can you go make sure that all the disasm are printed before this error? It' easier to debugging if that is correct

Comment (Zangetsu38) [reply src/emulator/shader/src/translator/alu.cpp]: no is needed, and velocityu have request me i remove this !, and here have 2 or 3 different src, so i have added this for can know quickely what exaclty is wrong

Comment (pent0) [reply src/emulator/shader/src/translator/alu.cpp]: Yeah, but don't print sources. It doesn't help anything for me though, and finding it in the SPIR-V code is useless.

If you insists to keep it,change them to src0 all the way (instead of vsrc0)

Comment (Zangetsu38) [reply src/emulator/shader/src/translator/alu.cpp]: hmm use "inst.opr.src0" get to much, and use just vsrc0 works good also, see on log few time so is good like that

Comment (Zangetsu38) [reply src/emulator/shader/src/translator/alu.cpp]: for one exemple just see here on other function
>[shader::usse::USSETranslatorVisitor::vtst]: Source not loaded (lhs: 0, rhs: 71)

so it said here lhs problem with 0, and rhs with 71

Comment (pent0) [reply src/emulator/shader/src/translator/alu.cpp]: ok
